src/Utilities/functions.jl

```diff
@@ -241,65 +246,51 @@ function canonical(f::MOI.ScalarAffineFunction)
     if iscanonical(f)
         f
     else
-        canonical!(MOI.ScalarAffineFunction(copy(f.terms), f.constant))
+        canonicalize!(MOI.ScalarAffineFunction(copy(f.terms), f.constant))
     end
 end
 
 function canonical(f::MOI.VectorAffineFunction)
     if iscanonical(f)
         f
     else
-        canonical!(MOI.VectorAffineFunction(copy(f.terms), f.constants))
+        canonicalize!(MOI.VectorAffineFunction(copy(f.terms), f.constants))
```

Comment:
Can use the `_constant` function to merge this and the previous `canonical`s.

Reply:
done (I'm using it in my implementation of Base.copy)